### DISCUSSION TITLE
How to not make Tablet in Osu Lazer correspond to a Part of The Game

### DISCUSSION THREAD

**Discussion by Nilshunt3r** [ppy/osu#17830] [Q&A]
When i want to play osulazer my Tablet Area Corresponds to a part of the screen Which is not like that in osu!

Clip of What Happens https://medal.tv/games/osu/clips/hQdN2BTR-UxZM/d1337VNH3wUX?invite=cr-MSxlNTksMjcyMzUyNjMs

**Comment by frenzibyte** [comment]
I'm not sure anything can be done with just this video. Please state your platform, whether you have a second display, and what part of your display you can move your tablet on (best with visuals, as I'm not even sure what you're doing in that video).

Also, can you try changing "Confine mouse cursor to window" to "Never"? (switch to borderless to disable confinement) That might be relevant.
